src/docs/sdk/rate-limiting.mdx

```diff
@@ -18,7 +18,7 @@ Each *quota_limit* has the form `retry_after:categories:scope:reason_code:namesp
     While these categories might look similar to the [envelope item types](/sdk/envelopes/#data-model), they are not identical, and have slight differences.
 - `scope`: The scope that this limit applies to. Can be ignored by SDKs.
 - `reason_code`: A unique identifier for the quota hinting at the rate limiting reason. Can be ignored by SDKs.
-- `namespaces`: Semicolon-separated list of metric namespace identifiers. This will only be present if the rate limit applies to the `metric_bucket` data category.
+- `namespaces`: Semicolon-separated list of metric namespace identifiers. This will only be present if the rate limit applies to the `metric_bucket` data category. If the namespace is not present, the back off applies to all metrics.
```

Comment:
```suggestion
- `namespaces`: Semicolon-separated list of metric namespace identifiers. This will only be present if the rate limit applies to the `metric_bucket` data category. If the namespace is not present, the backoff applies to all metrics.
```